foundation/console/down_command_test.go

```diff
@@ -0,0 +1,79 @@
+package console
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	mocksconsole "github.com/goravel/framework/mocks/console"
+	mocksfoundation "github.com/goravel/framework/mocks/foundation"
+)
+
+type DownCommandTestSuite struct {
+	suite.Suite
+}
+
+func TestDownCommandTestSuite(t *testing.T) {
+	suite.Run(t, new(DownCommandTestSuite))
+}
+
+func (s *DownCommandTestSuite) SetupSuite() {
+}
+
+func (s *DownCommandTestSuite) TearDownSuite() {
+}
+
+func (s *DownCommandTestSuite) TestSignature() {
+	expected := "down"
+	s.Require().Equal(expected, NewDownCommand(mocksfoundation.NewApplication(s.T())).Signature())
+}
+
+func (s *DownCommandTestSuite) TestDescription() {
+	expected := "Put the application into maintenance mode"
+	s.Require().Equal(expected, NewDownCommand(mocksfoundation.NewApplication(s.T())).Description())
+}
+
+func (s *DownCommandTestSuite) TestExtend() {
+	cmd := NewDownCommand(mocksfoundation.NewApplication(s.T()))
+	got := cmd.Extend()
+
+	s.Empty(got)
+}
+
+func (s *DownCommandTestSuite) TestHandle() {
+	app := mocksfoundation.NewApplication(s.T())
+	tmpfile := filepath.Join(os.TempDir(), "/down")
```

Comment:
In unit tests, use `t.TempDir()` instead of `os.TempDir()`.

`t.TempDir() `is preferred in unit tests because it creates a unique temporary directory for each test and automatically cleans it up after the test finishes. This helps prevent conflicts and side effects between tests, making your tests safer and more reliable.